src/app/lib/utilities/episodeAvailability/index.js

```diff
@@ -39,7 +39,7 @@ const getEpisodeAvailability = pageData => {
   const availableFrom = getEpisodeAvailableFrom(pageData);
   const availableUntil = getEpisodeAvailableUntil(pageData);
 
-  if (!availableUntil) {
+  if (!availableFrom || availableUntil < timeNow) {
```

Comment:
`availableUntil` will not be returned if the episode has expired

Reply:
Not always - there are some examples where `availableUntil` is not returned, but episode is still available - seems to be mostly Korean, learning english radio brand. 

If an episode is genuinely expired, versions is returned as an empty array, therefore both `availableFrom` and `availableUntil` are undefined, which is why the !availableFrom check is better to use for the expired check.

See https://github.com/bbc/simorgh/issues/7059#issuecomment-651207713 for analysis.

Reply:
No, I mean that `availableUntil < timeNow` will never apply as `availableUntil` will not be returned for expired episodes. It will be undefined.

Reply:
Gotcha. I couldn't find any examples of where `availableUntil < timeNow` anyway, so should I just remove and leave the `!availableFrom` check?